vaccine_feed_ingest/runners/tx/arcgis/normalize.py

```diff
@@ -0,0 +1,229 @@
+            phone = f"({source_phone[0:3]}) {source_phone[3:6]}-{source_phone[6:]}"
+            contacts.append(schema.Contact(phone=phone))
+        else:
+            logging.warning(f"Phone Number is unexpected size:{public_phone}")
+            contacts.append(schema.Contact(other=public_phone))
+
+    if site_attributes["WEBSITE"]:
+        contacts.append(schema.Contact(website=site_attributes["WEBSITE"]))
+    return contacts if contacts else None
+
+
+def _get_inventory(site_attributes: dict) -> List[schema.Vaccine]:
+    return [
+        schema.Vaccine(vaccine="jj", supply_level=site_attributes["JJ_AVAILABLE"]),
+        schema.Vaccine(
+            vaccine="pfizer", supply_level=site_attributes["PFIZER_AVAILABLE"]
+        ),
+        schema.Vaccine(
+            vaccine="pfizer_2", supply_level=site_attributes["PFIZER_AVAILABLE2"]
+        ),
+        schema.Vaccine(
+            vaccine="moderna", supply_level=site_attributes["MODERNA_AVAILABLE"]
+        ),
+        schema.Vaccine(
+            vaccine="moderna_2", supply_level=site_attributes["MODERNA_AVAILABLE2"]
+        ),
+    ]
+
+
+def _get_published_at(site_attributes: dict) -> Optional[str]:
+    update_date_str = site_attributes["LAST_UPDATE_VAC"].strip()
+    update_time_str = site_attributes["LAST_UPDATE_TIME_VAC"].strip()
+    try:
+        update_date = datetime.datetime.strptime(update_date_str, "%m/%d/%Y").date()
+        update_time = datetime.datetime.strptime(update_time_str, "%H:%M:%S").time()
+        combined_date = datetime.datetime.combine(update_date, update_time)
+        return combined_date.isoformat()
+    except ValueError as e:
+        logging.error(
+            f"Error parsing date update_date_str={update_date_str}, update_time_str={update_time_str}: {e}"
+        )
+        return None
+
+
+def _get_location(site) -> Optional[schema.LatLng]:
+    # A few records are missing the geometry key entirely
+    if "geometry" in site:
+        return schema.LatLng(
+            latitude=site["geometry"]["y"],
+            longitude=site["geometry"]["x"],
+        )
+    else:
+        return None
+
+
+def _get_normalized_location(site: dict, timestamp: str) -> schema.NormalizedLocation:
+    site_attributes = site["attributes"]
+    return schema.NormalizedLocation(
+        id=_get_id(site),
+        name=site_attributes["NAME"],
+        address=_get_address(site_attributes),
+        location=_get_location(site),
+        contact=_get_contacts(site_attributes),
+        languages=None,
+        opening_dates=None,
+        opening_hours=None,
+        availability=None,
+        inventory=_get_inventory(site_attributes),
+        access=None,
+        parent_organization=None,
+        links=None,
+        notes=None,
+        active=None,
+        source=schema.Source(
+            source="arcgis",
+            id=site_attributes["OBJECTID"],
+            fetched_from_uri="UNKNOWN",  # TODO where is this from? Hard to tell from fetch.yaml
```

Comment:
This should be `https://tdem.maps.arcgis.com/apps/webappviewer/index.html?id=3700a84845c5470cb0dc3ddace5c376b`. Sorry for the mystery URL - I forgot to include them in the issues for many `arcgis` normalizers.

Reply:
https://github.com/CAVaccineInventory/vaccine-feed-ingest/pull/134/commits/1bf88f731d68818a3af3242079fa2f5c2ee6c292